src/QMCWaveFunctions/TWFFastDerivWrapper.h

```diff
@@ -40,7 +40,7 @@ class TWFFastDerivWrapper
   using ValueVector = SPOSet::ValueVector;
   using GradVector  = SPOSet::GradVector;
 
-
+  inline TWFFastDerivWrapper(){};
```

Comment:
Compiler handles the default constructor. Having an empty constructor is not considered a good practice.
Use either
```
TWFFastDerivWrapper() = default;
```
or simply remove it.
If the intention is to move the compilation to cpp.
Use
```
TWFFastDerivWrapper(); # in *.h
TWFFastDerivWrapper() =  default; # in *.cpp
```

Reply:
That's fair.  I'll fix that.